libmamba/src/core/channel.cpp

```diff
@@ -242,51 +112,29 @@ namespace mamba
     util::flat_set<std::pair<std::string, std::string>>
     Channel::platform_urls(bool with_credential) const
     {
-        std::string base = location();
-        if (with_credential && token())
+        if (!url().package().empty())
         {
-            base = util::join_url(base, "t", *token());
+            return {};
```

Comment:
Is an empty set expected in that case?

Reply:
Yes. At least that is what I am aiming for. When specifiying a full URL to a package `https://mamba.org/somthing/pkg.conda` I am not assuming any repo/channel structure.

In the future, I'd like to better separate these two cases.